devtools/maven/src/main/java/io/quarkus/maven/CreateExtensionMojo.java

```diff
@@ -443,7 +446,10 @@ static String getJavaPackage(String groupId, String javaPackageInfix, String art
                 segments.add(segment);
             }
         }
-        return segments.stream().collect(Collectors.joining("."));
+        return segments.stream() //
```

Comment:
why those `//` at the end of the lines?

Reply:
To prevent the autoformatter to join the lines back :)